pyproject.toml

```diff
@@ -26,6 +26,13 @@ classifiers = [
 [project.urls]
 Homepage = "https://github.com/quixio/quix-streams"
 
+[project.optional-dependencies]
+all = [
+  "fastavro"
```

Comment:
Hey @quentin-quix , let's pin the min and max versions here to be safe, otherwise we may hit some backward compatibility issues when some changes are introduced to the `fastavro` package.
Like `fastavro>=1.8,<2.0` or similar.